mimic/model/nova_objects.py

```diff
@@ -915,6 +915,26 @@ def request_action(self, http_action_request, server_id):
                                          " while it is in task state other than active",
                                          http_action_request))
 
+        elif 'rebuild' in action_json:
+            image_ref = action_json['rebuild'].get('imageRef')
+            if not image_ref:
+                return dumps(bad_request("Could not parse imageRef from request.", http_action_request))
+            if server.status == 'ACTIVE':
+                server.image_ref = image_ref
+                server.status = 'REBUILD'
+                http_action_request.setResponseCode(202)
+                server.collection.clock.callLater(
+                    5.0,
+                    server.update_status,
+                    u"ACTIVE")
+                server_details = server.detail_json(absolutize_url)
+                server_details['adminPass'] = 'password'
+                return dumps({"server": server_details})
+            else:
+                return dumps(conflicting("Cannot 'rebuild' instance " + server_id +
+                                         " while it is in task state other than active",
+                                         http_action_request))
+
```

Comment:
I think you've got it from here, the API handler I was referring to is the handler for `'/v2/<string:tenant_id>/servers/<string:server_id>/action'` that is the only caller of the method.